terraform/aws/analytical-platform-data-production/github-airflow-cjs-dashboard-data/iam-roles.tf

```diff
@@ -0,0 +1,14 @@
+module "github_airflow_cjs_dashboard_data_iam_role" {
+  #checkov:skip=CKV_TF_1:Module is from Terraform registry
+
+  source  = "terraform-aws-modules/iam/aws//modules/iam-github-oidc-role"
```

Comment:
```suggestion
  source  = "terraform-aws-modules/iam/aws//modules/github-airflow-cjs-dashboard-data"
```

Reply:
@jacobwoffenden  is n't that the source module  as in [link](https://registry.terraform.io/modules/terraform-aws-modules/iam/aws/latest/submodules/iam-github-oidc-role)

Reply:
yes, i'm not doing well with this reading malarky